source/pattern-library/navigation/horizontal-navigation/index.md

```diff
@@ -14,6 +14,16 @@ layout: page-tabs
     </ul>
     <p>Consider using <a href="{{site.baseurl}}pattern-library/navigation/vertical-navigation/">vertical navigation</a> if you need a solution that can easily scale for a large amount of menu items and easily adapt to small screen sizes. Horizontal menus can also be made responsive but it usually requires a transformation from horizontal to vertical. Since vertical menus are already in this format, the transition from desktop to mobile is less disorienting.</p>
     <p>Jump to <a href="#example-overview-1">Standard Horizontal Navigation</a>, or <a href="#example-overview-2">Navbar Variations</a></p>
+    <h2 id="example-overview-2">Masthead Design</h2>
```

Comment:
I downloaded and ran the changes, but the jump-to links appear to be broken. For example, the id "example-overview-2" appears twice in the page.

<h2 id="example-overview-2">Masthead Design</h2>
and
<h2 id="example-overview-2">Navbar Variations</h2>